src/main/webapp/src/app/calendar-task/task.component.ts

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'calendar-tasks',
+  templateUrl: './task.component.html',
+  styleUrls: ['./task.component.css', '../common/growpod-page-styles.css'],
+})
+export class TaskComponent implements OnInit {
+  constructor() {}
+
+  ngOnInit(): void {}
+
+  /**
+   * Clear tasks before adding any new elements
+   */
+  clearTasks(): void {
+    const tasksContainer = document.getElementById('all-tasks');
+    while (tasksContainer.firstChild) {
```

Comment:
I chose to use the removeChild method because it is generally faster than setting innerHTML to an empty string.
Test: https://www.measurethat.net/Benchmarks/Show/34/0/innerhtml-vs-removechild